graylog2-web-interface/test/helpers/mocking/EventAndEventDefinitions_mock.ts

```diff
@@ -0,0 +1,381 @@
+import AggregationWidgetConfig from 'views/logic/aggregationbuilder/AggregationWidgetConfig';
+import Pivot from 'views/logic/aggregationbuilder/Pivot';
+import Series from 'views/logic/aggregationbuilder/Series';
+import SortConfig from 'views/logic/aggregationbuilder/SortConfig';
+import Direction from 'views/logic/aggregationbuilder/Direction';
+import { allMessagesTable, resultHistogram } from 'views/logic/Widgets';
+
+export const mockEventData = {
+  event: {
+    alert: true,
+    id: 'event-id-1',
+    event_definition_id: 'event-definition-id-1',
+    event_definition_type: 'aggregation-v1',
+    origin_context: null,
+    timestamp: '2023-03-02T13:43:21.266Z',
+    timestamp_processing: '2023-03-02T13:43:21.906Z',
+    timerange_start: '2023-03-02T13:42:21.266Z',
+    timerange_end: '2023-03-02T13:43:21.266Z',
+    streams: [
+      '002',
+    ],
+    source_streams: [
+      '001',
+    ],
+    message: 'message',
+    source: '',
+    key_tuple: [],
+    key: null,
+    priority: '2',
+    fields: [{}],
+    replay_info: {
+      timerange_start: '2023-03-02T13:42:21.266Z',
+      timerange_end: '2023-03-02T13:43:21.266Z',
+      query: 'http_method: GET',
+      streams: [
+        '001',
+      ],
+    },
+    group_by_fields: [{}],
+
+  } as Event,
+};
+
+export const mockEventDefinitionTwoAggregations:EventDefinition = {
+  _scope: 'DEFAULT',
+  id: 'event-definition-id-1',
+  title: 'Event Definition Title',
+  description: 'Test description',
+  updated_at: '2023-02-21T13:28:09.296Z',
+  priority: 2,
+  alert: true,
+  config: {
+    type: 'aggregation-v1',
+    query: 'http_method: GET',
+    query_parameters: [],
+    streams: [
+      '001',
+    ],
+    group_by: [
+      'field1',
+      'field2',
+    ],
+    series: [
+      {
+        id: 'count-field1',
+        function: 'count',
+        field: 'field1',
+      },
+      {
+        id: 'count-field2',
+        function: 'count',
+        field: 'field2',
+      },
+    ],
+    conditions: {
+      expression: {
+        expr: '||',
+        left: {
+          expr: '>',
+          left: {
+            expr: 'number-ref',
+            ref: 'count-field1',
+          },
+          right: {
+            expr: 'number',
+            value: 500.0,
+          },
+        },
+        right: {
+          expr: '<',
+          left: {
+            expr: 'number-ref',
+            ref: 'count-field2',
+          },
+          right: {
+            expr: 'number',
+            value: 8000.0,
+          },
+        },
+      },
+    },
+    search_within_ms: 60000,
+    execute_every_ms: 60000,
+  },
+  field_spec: {},
+  key_spec: [],
+  notification_settings: {
+    grace_period_ms: 60000,
+    backlog_size: 0,
+  },
+  notifications: [
+    {
+      notification_id: 'email_notification_id',
+      notification_parameters: null,
+    },
+  ],
+  storage: [
+    {
+      type: 'persist-to-streams-v1',
+      streams: [
+        '0002',
+      ],
+    },
+  ],
+};
```

Comment:
You could also move the fixtures into a JSON file and read them with `readJsonFixture`.